client/src/main/scala/com.gu.contentapi.client/ContentApiClient.scala

```diff
@@ -41,7 +41,8 @@ trait ContentApiClient {
   def get(url: String, headers: Map[String, String])(implicit context: ExecutionContext): Future[HttpResponse]
 
   /** Some HTTP headers sent along each CAPI request */
-  private def headers = Map("User-Agent" -> userAgent, "Accept" -> "application/x-thrift")
+  private def headers(retry: Int) =
+    Map("User-Agent" -> userAgent, "Accept" -> "application/x-thrift", "x-retry" -> s"$retry")
```

Comment:
Since a lot of those will be 0, can we make it an `Option[Int]` instead, and only add the header when there were actual retries?

Reply:
I'd rather see the zero to begin with to know that it's working. It's currently not coming through anywhere at all.